docs/source/services/services.rst

```diff
@@ -0,0 +1,246 @@
+.. _ref_rest:
+
+========
+Services
+========
+
+.. container:: left-col
+
+    ``TARDIS`` provides the possibility to start-up dedicated services along with its main process. Those services must
+    be decorated with the ``service`` decorator implemented in the ``cobald.daemon`` module. In order to allow dynamic
+    creation of those ``services`` via ``yaml`` tags, the service needs to be decorated with the
+    ``@yaml_tag(eager=True)`` decorator as well. The services are then dynamically created and started during the
+    ``yaml`` parsing of the configuration. All services are configured within the ``Services`` section of the
+    ``TARDIS`` configuration using ``yaml`` tags. Using multiple services is supported by using a separate ``yaml``
+    MappingNode per service.
+
+.. container:: content-tabs right-col
+
+    .. rubric:: Example implementation
+
+    .. code-block:: python
+
+        from cobald.daemon import service
+        from cobald.daemon.plugins import yaml_tag
+
+
+        @service(flavour=asyncio)
+        @yaml_tag(eager=True)
+        class ExampleService(object):
+            def __init__(option_1):
+                ...
+            async def run():
+                ...
+
+    .. rubric:: Example configuration
+
+    .. code-block:: yaml
+
+        Services:
+            ExampleService:
+                !ExampleService
+                option_1: my_option_1
+            Service_2:
+                !TardisService2
+                option_123: my_option_123
+
+REST Service
+------------
+
+.. content-tabs:: left-col
+
+    The :py:class:`~tardis.rest.service.RestService` provides a REST API to access the information stored about managed
+    ``Drones`` inside the :py:class:`~tardis.plugins.sqliteregistry.SqliteRegistry` plugin. Currently access is read
+    only and limited to either list all managed resources or to get the state of a particular resource. All supported
+    REST API calls and their parameters are described in REST API documentation available on
+    ``http://<hostname>:<port>/docs`` after starting the service. The REST service is using JSON Web Token (JWT) and
+    OAuth2 scopes for authentication and authorization.
+
```

Comment:
I think it should be mentioned that this could be extended as mentioned in the [discussion](https://github.com/MatterMiners/tardis/pull/183#issuecomment-1010081705). Maybe add an outlook that it could be possible to make configuration changes or delete resources.

Reply:
Thanks for the remark. Done!